src/main/java/nl/quintor/solitaire/models/card/Card.java

```diff
@@ -129,4 +129,11 @@ public String toString() {
     public String toShortString() {
         return shortDescription;
     }
+
+    //returns true if the card is red, false if the card is black
+    public boolean isRed(){
+        //a boolean return value for each suit [clubs, diamonds, hearts, spades, joker]
```

Comment:
Cool solution but a little hard to read and there is a helper function already in the CardMoveChecks class.